bootstrap/app.php

```diff
@@ -11,9 +11,7 @@
 |
 */
 
-$app = new Illuminate\Foundation\Application(
-    $_ENV['APP_BASE_PATH'] ?? dirname(__DIR__)
-);
+$app = new Illuminate\Foundation\Application($_ENV['APP_BASE_PATH'] ?? dirname(__DIR__));
```

Comment:
I'm conflicted on modification to files in `bootstrap/` as these are part of the framework and not really the app. Perhaps the `bootstrap/` files can be excluded from formatting?

Reply:
So you want 120 chars but you don't want 120 on every single file?

This file is on your app and it will not be changed/updated by the framework anymore, they are only "recommendation" and I don't believe format is what they should recommend, the framework lies on vendor/laravel/framework. We are free to update these files as we please, so what happens if we do? We keep the format or we update to the new format?

Reply:
Same below

Reply:
@antonioribeiro my concern is introducing changes to _framework_ files, not application files. Modifying these files might introduce merge conflicts when upgrading the framework between versions, not the application itself.

Is not _what I want_ vs _what I don't want_. I thought I clarified that in my first comment on this about framework vs app codebase.

Reply:
Files are ignored on Prettier now, @luislavena

Reply:
Maybe you should consider keeping Laravel's repository history on the codebase so we can merge/rebase from their, and we can also add all Laravel original files to the .prettierignore and not having any changes applied to these files. 

Not really sure what files you want and you don't want updated, like, what's the difference between bootstrap/app.php and app/Providers/RouteServiceProvider.php one can be formatted but not the other one, or should we keep both unformatted?